spec/query_spec.rb

```diff
@@ -1039,6 +1039,38 @@ def to_hash
     end
   end
 
+  describe '#to_string' do
+    it 'serializes' do
```

Comment:
I think this suite is meant for functional tests.

Reply:
I noticed that; it's odd that some drivers seem to only have functional tests, where others have serialization.

I take it you prefer I rewrite as a functional test?

Reply:
If I not mistake all drivers has functional tests, but not all has serializations

Reply:
Serialization makes more sense to me, in that the semantics of the language is defined and tested in the database. The clients need only conform to the wire protocol.

That said, I'll rewrite these as functional tests.

Reply:
I'm afraid only conform to wire protocol isn't not enough, we catch a lot of edge cases while developing the drivers because of the functional tests, those tests force us to really utilize the api and see if the functions fit together. One exemple was recursive references, I found out there wasn't a way to paginate over nested objects when I had to "paginate" those objects.